.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
Can you gitignore this?

Reply:
I'd rather not - if you're running helsing's nix-template (or direnv in general) that will automatically start the devshell with all of the project's dependencies + dev tools when you enter the directory.

Reply:
Yes, this is helsing nix-template specific which is why I asked everyone that had these in their PR until now (happened multiple times) to remove them as they don't serve a broader usage to the general public!

I am not convinced that local environment settings, such as your shell setup and QoL features, IDE settings or alike should live in an open source repository! 🙈 There are just too many different tools (for example for the env management: direnv, ondir, autoenv, shadowenv etc) for this and you clutter the repository over time if you don't have a strict policy on this.

I will put some time to write a `CONTRIBUTING.md` soon